apps/portals/b2ai.standards/src/components/ChallengesCardDeck.tsx

```diff
@@ -1,59 +1,57 @@
 import {
   TABLE_IDS,
-  CHALLENGES_TABLE_COLUMN_NAMES,
-  dataSql,
-  DST_TABLE_COLUMN_NAMES,
   ORG_TABLE_COLUMN_NAMES,
+  ORG_TABLE_JSON_COLUMNS,
+  GC_ORG_IDS,
 } from '@/config/resources'
-import { Query, QueryBundleRequest } from '@sage-bionetworks/synapse-types'
+import { GCInfo } from '@/config/GrandChallengeResources'
+import {
+  QueryBundleRequest,
+  ColumnSingleValueQueryFilter,
+  ColumnSingleValueFilterOperator,
+} from '@sage-bionetworks/synapse-types'
 import { CardDeck } from 'synapse-react-client/components/CardDeck/CardDeck'
 import { CardDeckCardProps } from 'synapse-react-client/components/CardDeck/CardDeckCardProps'
 import useGetQueryResultBundle from 'synapse-react-client/synapse-queries/entity/useGetQueryResultBundle'
-import {
-  ErrorBanner,
-  getFileHandleAssociation,
-  SynapseConstants,
-} from 'synapse-react-client'
+import { ErrorBanner, SynapseConstants } from 'synapse-react-client'
 import { getFieldIndex } from 'synapse-react-client/utils/functions/queryUtils'
-import { useGetEntity } from 'synapse-react-client/synapse-queries/entity/useEntity'
-import { ImageFileHandle } from 'synapse-react-client/components/widgets/ImageFileHandle'
 
 /**
  * Card view of challenges for the home page
  */
 
-function createExplorePageLink(query: Query): string {
-  return `/Explore?QueryWrapper0=${encodeURIComponent(JSON.stringify(query))}`
-}
-
-type useTableFetchProps = {
-  entityId: string
-  columns: string[]
-  sql?: string
-  shouldRun?: boolean
-}
-
-export function useTableFetch({
-  entityId,
-  columns, // required
-  sql, // if filters or aliases or anything are needed, pass in whole sql query
-  shouldRun = true, // if calling before ready, send false
-}: useTableFetchProps) {
-  if (!sql) {
-    sql = `SELECT ${columns.join(', ')} FROM ${entityId}`
-  }
+export function ChallengesCardDeck() {
+  const additionalFilters: ColumnSingleValueQueryFilter[] = [
```

Comment:
nit: this could just be the generic `QueryFilter`
```suggestion
  const additionalFilters: QueryFilter[] = [
```